geomstats/_backend/tensorflow/linalg.py

```diff
@@ -13,6 +13,10 @@
 sqrtm = tf.linalg.sqrtm
 
 
+def eig(*args, **kwargs):
```

Comment:
Do you want to use _raise_not_implemented_error here?

Reply:
For the moment, yes, because the pytorch backend didn't provide an `eig` implementation so far. Most of the functions which currently raise a `NotImplementedError` I plan on implementing (where possible) in the course of the next PR adding unit tests for backend functions.